demo/trusted_grub/README.md

```diff
@@ -0,0 +1,21 @@
+# TrustedGRUB
+
+A demonstration of trusted boot using `TrustedGRUB`.
+
+This demonstration will be depreciated when work is complete on keylimes
```

Comment:
It could be, but from what I can tell this is not really demoing anything in keylime, its just TrustedGRUB working on it's own. Unless I have read things wrong, keylime does not interact with  TrustedGRUB? I originally thought this might have been a case of Keylime would remotely attest a system booted with TrustedGRUB.

Perhaps I have understood this wrong, if so , I am keen to see this working as it's a key feature, but as it stands it's a little misleading as it suggests (or it did to me at least) like it's going to demo keylime working with some sort of trusted boot measurement or signal a nodes integrity. 

I mentioned rhboot, as my hope is that we develop some sort of remote attest or reading of a Stored Measurement Log.

Reply:
The TrustedGRUB demo we usually give actually goes much more in depth than that, where Keylime detects when an unauthorized kernel boots up and refuses to provision the machine.  

TrustedGRUB writes out kernel- and configuration-related measurements to PCR-10 and PCR-11 (see [TrustedGRUB2#13-measurements-in-short](https://github.com/Rohde-Schwarz-Cybersecurity/TrustedGRUB2#13-measurements-in-short)).  Note that [our patch](https://github.com/keylime/python-keylime/blob/master/patches/trustedgrub-patch.txt) moves PCR-10 to PCR-14, since we use PCR-10 for IMA.  

An easy way to set this up is just to have two kernel versions that are bootable on the machine (e.g., after a kernel upgrade, Fedora and Ubuntu usually keep a few older versions available as a fallback).  If you boot into the "good"/latest kernel and write down its PCR-11 value, then you can use this to enforce only allowing that configuration to be provisioned in Keylime (via the `--tpm_policy` tenant option).  You can see the default TPM policy at [keylime.conf#L259](https://github.com/keylime/python-keylime/blob/master/keylime.conf#L259), which just makes sure that PCR-15 and -22 are untouched. 

Using this, you can boot into the "good" kernel and show you can provision Keylime correctly, then reboot into one of the "bad" kernels and show that Keylime refuses to provision the machine (since PCR-11 should be different).

Reply:
ok, so this does sound like its certainly worth keeping. We need to get this documented (nudge , nudge, wink, wink) as it sounds like we have the set up part in place (using the demo_setup.sh) , but some specific CLI calls are needed from the tenant `--tpm_policy`.

May I create an issue for you to outline a set up example, I will then replay the same steps and produce some Docs? This patch can then be updated with what we have between us?